rakefile-dotnet.rb

```diff
@@ -0,0 +1,373 @@
+require 'albacore'
+
+msbuild_command = "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe"
+if !File.file?(msbuild_command)
+  raise "MSBuild not found"
+end
+
+nuget_command   = ".nuget/nuget.exe"
+gitlink_command = "packages/gitlink.2.3.0/lib/net45/GitLink.exe"
+xunit_command   = "packages/xunit.runner.console.2.0.0/tools/xunit.console.exe"
+
+solution        = "FakeItEasy-NetCore.sln"

```

Comment:
> Can one not build .xproj from within a .sln using MSBuild?

Maybe because of a bug in dotnet where it doesn't work well with the custom nuget restore package directory.